package.json

```diff
@@ -24,18 +24,6 @@
     "@balancer-labs/sor": "github:balancer-labs/balancer-sor#john/v1-consideringFees-package",
     "@balancer-labs/sor2": "github:balancer-labs/balancer-sor#john/ui-629-sor-integrate-sdk-maths",
     "@ensdomains/content-hash": "^2.5.3",
-    "@ethersproject/abi": "^5.0.1",
-    "@ethersproject/abstract-signer": "^5.3.0",
-    "@ethersproject/address": "^5.0.1",
-    "@ethersproject/bignumber": "^5.2.0",
-    "@ethersproject/bytes": "^5.0.5",
-    "@ethersproject/constants": "^5.0.1",
-    "@ethersproject/contracts": "^5.0.1",
-    "@ethersproject/hash": "^5.0.6",
-    "@ethersproject/providers": "^5.0.12",
-    "@ethersproject/strings": "^5.0.5",
-    "@ethersproject/units": "^5.0.1",
-    "@ethersproject/wallet": "^5.0.1",
```

Comment:
These are not needed if you import the main package. Any references to them in the code still works.

Reply:
Awesome... I think we can just import everything from `ethers` then right?
So like instead of:
`import { formatUnits } from '@ethersproject/units';`
You would write:
`import { formatUnits } from 'ethers/lib/utils';`

Reply:
Yes but it seems you can still use the @ethersproject syntax with just the main `ethers` package installed. What would be the benefit in going through the project and replacing all the imports?